### PR TITLE
PE-1137: Localize newly added strings

### PR DESCRIPTION
**Pull request by matibat**
This PR takes the newly introduced strings from [PE-1105](https://github.com/ardriveapp/ardrive-web/pull/401) (child of PE-815), [PE-1003](https://github.com/ardriveapp/ardrive-web/pull/385) and [PE-1121](https://github.com/ardriveapp/ardrive-web/pull/407) to apply to them internationalization

### CODE REVIEW

File: lib/components/create_manifest_form.dart [hunk 1]
```diff
@@ -76,38 +86,39 @@ class CreateManifestForm extends StatelessWidget {
...
               ],
             );
 
         if (state is CreateManifestWalletMismatch) {
           Navigator.pop(context);
           return errorDialog(
-            errorText:
-                'Provided wallet has unexpectedly changed during manifest creation...',
+            errorText: appLocalizationsOf(context)
+                .manifestTransactionUnexpectedlyFailed,
```
Comment (karlprieb) [inline lib/components/create_manifest_form.dart:98]: This is not the same of `manifestTransactionUnexpectedlyFailed`

File: lib/components/create_manifest_form.dart [hunk 2]
```diff
@@ -156,26 +169,28 @@ class CreateManifestForm extends StatelessWidget {
...
               actions: [
                 TextButton(
                     onPressed: () => Navigator.pop(context),
-                    child: Text('CANCEL')),
+                    child: Text(appLocalizationsOf(context).cancelEmphasized)),
                 ElevatedButton(
                   onPressed: () => readCubitContext.chooseTargetFolder(),
-                  child: Text('NEXT'),
+                  child: Text(
+                    appLocalizationsOf(context).continueEmphasized,
```
Comment (karlprieb) [inline lib/components/create_manifest_form.dart:192]: We will change `NEXT` for `CONTINUE`?

File: lib/components/create_manifest_form.dart (hunk 1 above)
Comment (matibat) [reply lib/components/create_manifest_form.dart]: Good catch! aebdfd47e516161e7dcc406214199e5aa6c4bb0c

File: lib/components/create_manifest_form.dart (hunk 2 above)
Comment (matibat) [reply lib/components/create_manifest_form.dart]: aebdfd47e516161e7dcc406214199e5aa6c4bb0c

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -210,6 +210,10 @@
   "@next": {
     "description": "Action of going to the next page"
   },
+  "nextEmphasized": "NEXT",
+  "@nextEmphasized": {
+    "description": "Next strep"
```
Comment (fedellen) [inline lib/l10n/app_en.arb:215]:
```suggestion
    "description": "Next step"
```

**Review by karlprieb — APPROVED** [review]

**Review by fedellen — APPROVED** [review]

**Review by jdaev — APPROVED** [review]

**Review by fedellen — APPROVED** [review]

**Review by jdaev — APPROVED** [review]
👍🏽

**Pull request merged**